sdk/program/src/message/versions/v0/mod.rs

```diff
@@ -0,0 +1,374 @@
+use crate::{
+    hash::Hash,
+    instruction::CompiledInstruction,
+    message::{MessageHeader, MESSAGE_VERSION_PREFIX},
+    pubkey::Pubkey,
+    sanitize::{Sanitize, SanitizeError},
+    short_vec,
+};
+
+mod loaded;
+
+pub use loaded::*;
+
+/// Address table lookups describe an on-chain address lookup table to use
+/// for loading more readonly and writable accounts in a single tx.
+#[derive(Serialize, Deserialize, Default, Debug, PartialEq, Eq, Clone, AbiExample)]
+#[serde(rename_all = "camelCase")]
+pub struct MessageAddressTableLookup {
+    /// Address lookup table account key
+    pub account_key: Pubkey,
```

Comment:
How is the account_key used? Where do we actually store the table?

Reply:
There are many such tables (users can create their own as they please) and they are all on-chain accounts so they are identified by this account key.